modules/ROOT/pages/sync/provisioning.adoc

```diff
@@ -29,6 +31,7 @@ If the Postgres password was generated, retrieve the password using:
 ----
 oc describe dc postgres-data-sync -n <myproject> | grep POSTGRESQL_PASSWORD
 ----
+Wait for the database provisioning to finish before executing this command.
```

Comment:
@pmacko1  didn't we have something like this recently? This warning should be before the command.

Reply:
Yep, in metrics, see comment from @aidenkeating 
https://docs.google.com/document/d/1q49WNxWQPnUWF6_97ot305coDoI3sgYJ9QUz89JKXzk/edit
and yours in data sync
https://docs.google.com/document/d/1L5o7L715Q_pO0y3VGY0enQ7dqdmAA53qkqAV7IVLTfg/edit#

Reply:
https://github.com/aerogear/mobile-docs/issues/459